src/newton_solver.cpp

```diff
@@ -75,7 +75,11 @@ std::unique_ptr<NewtonSolver> NewtonSolver::getSolver(realtype *t, AmiVector *x,
     solver->dampingFactorMode = simulationSolver.getNewtonDampingFactorMode();
     solver->dampingFactorLowerBound =
         simulationSolver.getNewtonDampingFactorLowerBound();
-    solver->numlinsteps.resize(simulationSolver.getNewtonMaxSteps(), 0.0);
+    if (simulationSolver.getLinearSolver() == LinearSolver::SPBCG) {
+        solver->numlinsteps.resize(simulationSolver.getNewtonMaxSteps(), 0.0);
```

Comment:
```suggestion
        solver->numlinsteps.resize(simulationSolver.getNewtonMaxSteps(), 0);
```

Was already there before, but as this is an `int` array, using double seems unnecessarily confusing...

Reply:
ah, true, sorry...